src/axom/slic/core/Logger.cpp

```diff
@@ -16,8 +16,21 @@ namespace axom
 {
 namespace slic
 {
-Logger* Logger::s_Logger = nullptr;
-std::map<std::string, Logger*> Logger::s_loggers;
+using Loggermap = std::map<std::string, Logger*>;
+
+//------------------------------------------------------------------------------
+Loggermap& getLoggers()
```

Comment:
Since it's not in the function name and this is a free function that's limited to the cpp file, could you please add a comment that this is a singleton (for `getLoggers` and `getLogger`)?

Reply:
done.